Makefile

```diff
@@ -30,7 +30,7 @@ test-sanity:
 
 .PHONY: test-e2e
 test-e2e:
-	go test -v ./tests/e2e/...
+	ginkgo -v ./tests/e2e/...
```

Comment:
Im wondering why need to replace `go test` with `ginkgo`?

Reply:
I need to enable Ginkgo's verbosity *and* I also need to run the test with admin privileges due to the `mount` command. For some reason, executing `go test -ginkgo.v` with `sudo` didn't work for me (I think it's related to `go modules`), so I called `ginkgo` directly.

In any case, I improved these bits by compiling the test and then calling it with `sudo`.